### ISSUE TITLE
SSH command responded with a non-zero exit status.

### ISSUE THREAD

**Issue by MKLau** [ProvTools/encapsulator#25]
Virtualbox opened but eventually stalled and tried to gracefully shutdown. Here's the output:

```
hermes:example hermes$ encapsulator --encapsulate matt/example messycode.R fig1_biplot.png
* installing *source* package ‘provR’ ...
Warning in as.POSIXlt.POSIXct(x, tz) :
  unknown timezone 'zone/tz/2018c.1.0/zoneinfo/America/New_York'
** R
** inst
** tests
** preparing package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded
* DONE (provR)
Script written to .matt/example/workspace/fig1_biplot.png.R
Saved input ../.matt/example/workspace/clean/data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.encapsulator/.mkl/messy_clean/workspace/.mkl/messy_clean2/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.encapsulator/.mkl/messy_clean/workspace/.mkl/messy_clean2/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.encapsulator/.mkl/messy_clean/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.encapsulator/.mkl/messy_clean/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.encapsulator/.mkl/messy_clean2/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.encapsulator/.mkl/messy_clean2/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.encapsulator/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.encapsulator/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean/workspace/.encapsulator/.mkl/messy_clean/workspace/.mkl/messy_clean2/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean/workspace/.encapsulator/.mkl/messy_clean/workspace/.mkl/messy_clean2/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean/workspace/.encapsulator/.mkl/messy_clean/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean/workspace/.encapsulator/.mkl/messy_clean/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean/workspace/.encapsulator/.mkl/messy_clean2/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean/workspace/.encapsulator/.mkl/messy_clean2/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean/workspace/.encapsulator/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean/workspace/.encapsulator/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean/workspace/.mkl/messy_clean2/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean/workspace/.mkl/messy_clean2/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean2/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messy_clean2/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.encapsulator/.mkl/messy_clean/workspace/.mkl/messy_clean2/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.encapsulator/.mkl/messy_clean/workspace/.mkl/messy_clean2/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.encapsulator/.mkl/messy_clean/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.encapsulator/.mkl/messy_clean/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.encapsulator/.mkl/messy_clean2/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.encapsulator/.mkl/messy_clean2/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.encapsulator/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.encapsulator/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean/workspace/.encapsulator/.mkl/messy_clean/workspace/.mkl/messy_clean2/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean/workspace/.encapsulator/.mkl/messy_clean/workspace/.mkl/messy_clean2/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean/workspace/.encapsulator/.mkl/messy_clean/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean/workspace/.encapsulator/.mkl/messy_clean/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean/workspace/.encapsulator/.mkl/messy_clean2/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean/workspace/.encapsulator/.mkl/messy_clean2/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean/workspace/.encapsulator/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean/workspace/.encapsulator/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean/workspace/.mkl/messy_clean2/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean/workspace/.mkl/messy_clean2/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean2/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/.mkl/messy_clean2/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/messycode/.mkl/messycode/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/projects/2016/july_biomass_survey.csv for script fig1_biplot.png.R
Saved input ../.matt/example/workspace/projects/2016/old_data/july_biomass_survey.csv for script fig1_biplot.png.R
Generated script for output fig1_biplot.png
In directory /Users/hermes/Labs/HF/projects/e2ep/projects/encapsulator/example/.matt/example...
Encapsulator will attempt to install the following packages:
Packages
--------
base v3.4.2
gdata v2.18.0
lattice v0.20-35
permute v0.9-4
txtplot v1.0-3
vegan v2.4-6
Provision script is ready...
getting your capsule ready, be patient...
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'jhcook/fedora25'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'jhcook/fedora25' is up to date...
==> default: There was a problem while downloading the metadata for your box
==> default: to check for updates. This is not an error, since it is usually due
==> default: to temporary network problems. This is just a warning. The problem
==> default: encountered was:
==> default: 
==> default: The requested URL returned error: 404 Not Found
==> default: 
==> default: If you want to check for box updates, verify your network connection
==> default: is valid and try again.
==> default: Setting the name of the VM: matt_example
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
[default] GuestAdditions versions on your host (5.2.6) and guest (5.1.22) do not match.
==> default: Checking for guest additions in VM...
    default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.
    default: 
    default: Guest Additions Version: 5.1.22
    default: VirtualBox Version: 5.2
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

`bash -c 'type -p dnf || type -p yum'` install -y kernel-devel-`uname -r` gcc dkms make perl bzip2

Stdout from the command:



Stderr from the command:



==> default: Attempting graceful shutdown of VM...
Your capsule should be visible in the virtualbox interface.
WARNING: We are sorry due to remote API change, capsule need to be manually uploaded to: https://app.vagrantup.com
hermes:example hermes$ ls
R_clean		clean		fig2.R		graph.dot	messycode.R	out.txt		sourced.R	workspace
Rplots.pdf	cleansrc	figures_1	graph.svg	messycode_ddg	projects	tmp.pdf		wspc
analysis_forest	fig1.R		figures_2	messycode	new		prov		tmp.png
```

**Comment by MKLau** [comment]
![screen shot 2018-03-07 at 4 56 49 pm](https://user-images.githubusercontent.com/5342857/37121088-2a5feae6-222a-11e8-8679-9a972f514f38.png)

**Comment by tfjmp** [comment]
It is getting late here, but will have a look tomorrow. My assessment is that the version of virtualbox/vagrant/guest vm are incompatible. That is easy to fix, but is likely to reappear as those software get updated. Not sure if anything can be done except maintaining the tool.

**Comment by MKLau** [comment]
Thanks, Thomas, definitely don't let this keep you up. This is not high priority. Chris's issue definitely tops mine. My machine's super old anyway, so I'm probably at the lagging edge of software support.

**Comment by MKLau** [comment]
This is now fixed with v 0.1.1.
